src/games/strategy/net/UniversalPlugAndPlayHelper.java

```diff
@@ -23,7 +23,8 @@
   private int m_port = 3300;
   private InetAddress m_local = null;
   private InternetGatewayDevice m_device = null;
-  private static UniversalPlugAndPlayHelper s_lastInstance = null;
+  @SuppressWarnings("unused")
+private static UniversalPlugAndPlayHelper s_lastInstance = null;
```

Comment:
formatting still needs to be applied.
Also pls see about removing this variable too instead of suppressing the warning.